src/controllers/movies/movietrailers.js

```diff
@@ -7,7 +7,7 @@ define(["layoutManager", "loading", "events", "libraryBrowser", "imageLoader", "
             var pageData = data[key];
 
             if (!pageData) {
-                pageData = data[key] = {
+                pageData = {
```

Comment:
any reason for this?

Reply:
Unless I'm mistaken, what it does is basically assign data[key] to pageData, then checking if pageData exists and, if not, assign that new object to pageData and data[key], but then data[key] goes unused for the rest of the file.

Reply:
But it would get used on next function call? Typical memoizing pattern, no?